include/updatehub.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
```

Comment:
We need a different location for this header, but not sure what the best location for it.

Reply:
Will raise this in our Thur Dev-review meeting to get input on what the path should be.

Reply:
@galak did you figure where do you guys wish this to be moved?

I'd like to get this merged soon and this seems to be the last open item.